workflows/phaseimpute/main.nf

```diff
@@ -97,31 +99,44 @@ workflow PHASEIMPUTE {
     if (params.steps.split(',').contains("simulate") || params.steps.split(',').contains("all")) {
         // Output channel of simulate process
         ch_sim_output = Channel.empty()
+        // Set truth channel
+        ch_input_validate_truth = ch_input_sim
```

Comment:
Why is the `ch_input_validate_truth` here? Shouldn't this be part of `--validate`?

Reply:
It's to rewrite the input of the validate step when both the step simulate and validate is selected.
But I might have made a mistake here. I will update it.